careers/game/careersGameEngine.py

```diff
@@ -622,8 +623,11 @@ def create(self, edition, installationId, game_type, points, game_id=None) -> Co
         self._game_state = self._careersGame.game_state
         self._gameId = self._careersGame.gameId         # the gameId includes the installationId
         self._logfile_filename = f'{self.gameId}_{self._careersGame.edition_name}'
-        self._logfile_folder = "/data/log"    # TODO put in Environment
-        self._gamefile_folder = "/data/games"
+        
```

Comment:
Not sure if this is the best way to do this....but it works locally AND in docker